samples/client/petstore/php/OpenAPIClient-php/lib/Api/UserApi.php

```diff
@@ -1775,6 +1827,10 @@ public function updateUserWithHttpInfo($username, $user)
                 );
             }
 
+            $responseBody = $response->getBody();
+            switch($statusCode) {
+            }
+
```

Comment:
Shall the generator not generate these 4 lines if there's no response (void return type in the function call)?

Reply:
You're right - fixed